dlib/simd/simd8f_vec.h

```diff
@@ -0,0 +1,251 @@
+// Copyright (C) 2013  Davis E. King (davis@dlib.net)
+// License: Boost Software License   See LICENSE.txt for the full license.
+#ifndef DLIB_sIMD8F_Hh_
```

Comment:
This whole file looks like a copy of simd8f.h.  Can't we just #include simd8f.h?